src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
If the dpdk value has to be the same as the VM memory, then the value needs to be read from the hypervisor.args to fetch the memory that has been configured before substituting in DPDK_CMDLINE_OBJ instead of hardcoding it for now.